docs/design/template.md

```diff
@@ -0,0 +1,53 @@
+# Design proposal template `<replace with your proposal's title>`
+
+_Note_: The preferred style for design documents is one sentence per line.
+*Do not wrap lines*.
+This aids in review of the document as changes to a line are not obscured by the reflowing those changes caused and has a side effect of avoiding debate about one or two space after a period.
+
+_Note_: The name of the file should follow the name pattern `<short meaningful words joined by '-'>_design.md`, e.g:
+`listener-design.md`.
```

Comment:
Should "listener-design.md" be "listener_design.md"? Or should the rule be "<short meaningful words joined by '-'>-design.md"?

Reply:
on OADP we seem to not follow this rule (have with dash, underscore or no suffix)

since we are in `design` folder, I would say we do not need suffix

but I do not have any preferences

Reply:
I don't have a preference either, I just want the example and the rule to match. Taking out the whole note sounds even better, if no one is paying attention to it anyway.

Reply:
done, removed the sentence